scripts/Calibration/tube.py

```diff
@@ -377,33 +400,39 @@ def calibrate(ws, tubeSet, knownPositions, funcForm, **kwargs):
         selectedTubes = tubeSet
         tubeSet = TubeSpec(ws)
         tubeSet.setTubeSpecByStringArray(selectedTubes)
-    elif not isinstance(tubeSet,TubeSpec):
-        raise RuntimeError("Wrong argument tubeSet. It must be a TubeSpec or a string that defines the set of tubes to be calibrated. For example: WISH/panel03")
+    elif not isinstance(tubeSet, TubeSpec):
+        raise RuntimeError(
+            "Wrong argument tubeSet. It must be a TubeSpec or a string that defines the set of tubes to be calibrated."
+            "For example: WISH/panel03")
 
     # check the known_positions parameter
     # for old version compatibility, it also accepts IdealTube, eventhough
     # they should only be used internally
-    if not (isinstance(knownPositions, list) or
-            isinstance(knownPositions, tuple) or
-            isinstance(knownPositions, numpy.ndarray)):
-        raise RuntimeError("Wrong argument knownPositions. It expects a list of values for the positions expected for the peaks in relation to the center of the tube")
+    if not (isinstance(knownPositions, list) or isinstance(knownPositions, tuple) or isinstance(knownPositions, numpy.ndarray)):
+        raise RuntimeError(
+            "Wrong argument knownPositions. It expects a list of values for the positions expected for the peaks in"
+            "relation to the center of the tube")
     else:
         idealTube = IdealTube()
         idealTube.setArray(numpy.array(knownPositions))
 
-
-    #deal with funcForm parameter
+    # pylint: disable = raising-bad-type
+    # deal with funcForm parameter
```

Comment:
it seems that `raising-bad-type` error is not being produced because `RuntimeError` is being utilised instead of `ValueError`